src/webview/memory-webview-view.tsx

```diff
@@ -177,6 +184,12 @@ class App extends React.Component<{}, MemoryAppState> {
         const columns = isVisible ? await columnContributionService.show(id, this.state) : columnContributionService.hide(id);
         this.setState(prevState => ({ ...prevState, columns }));
     }
+
+    protected toggleFreezeButton = (): void => { this.doToggleFreezeButton(); };
+    protected doToggleFreezeButton(): void {
+        this.setState(prevState => ({ ...prevState, isFrozen: !prevState.isFrozen }));
+    }
```

Comment:
These names sound more like they toggle the visibility of the button rather than the state itself.